TypeCobol/Compiler/Sql/Model/SavePointClause.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.IO;
+using TypeCobol.Compiler.CodeElements;
+
+namespace TypeCobol.Compiler.Sql.Model
+{
+    public class SavePointClause : SqlObject
+    {
+        private SymbolReference SavePointName { get; }
+        public SavePointClause(SymbolReference savePointName)
+        {
+            SavePointName = savePointName;
+        }
+        protected override bool VisitSqlObject(ISqlVisitor visitor)
+        {
+            return visitor.Visit(this);
+        }
+        protected override void DumpContent(TextWriter output, int indentLevel)
+        {
+            DumpProperty(output, nameof(SavePointName), SavePointName, indentLevel);
```

Comment:
Yes this is correct. At first I thought it was wrong because we have a different code for `TableViewCorrelationName.DumpContent` but this is cleaner to do it like that and let the generic `DumpProperty` do the work. Can you update `TableViewCorrelationName.DumpContent` with a similar code please ?